pkg/controller/cluster/cluster_reconcile_test.go

```diff
@@ -27,21 +28,38 @@ import (
 )
 
 // testSetupClient returns a client containing all objects in objs
-func testSetupClient(objs []runtime.Object) (client.Client, *runtime.Scheme) {
+func testSetupClient(objs map[schema.GroupVersion][]runtime.Object) (client.Client, *runtime.Scheme) {
```

Comment:
why this change to a map? It seems to just bloat the usages

We probably could just load our scheme with
```go
s := runtime.NewScheme()
runtime.Must(synv1alpha1.SchemeBuilder.AddToScheme(s))
runtime.Must(appsv1.SchemeBuilder.AddToScheme(s)) // whatever else needed
```
since I consider launching an operator without our schemes loaded like a compilation error: it should just refuse to start up.

Reply:
Objects need to be added along with ther scheme. This becomes relevant in #140. In there, I had the situation that Roles were associated both with `syn.tools` and `rbac.authorization.k8s.io`. The client fails then to load the object due to abiguity.